linkerd/app/test/src/connect.rs

```diff
@@ -46,15 +51,15 @@ impl<E: fmt::Debug> Connect<E> {
         }
     }
 
-    pub fn endpoint_future(
+    pub fn endpoint(
         self,
         endpoint: impl Into<SocketAddr>,
-        f: impl (FnMut(E) -> ConnectFuture) + Send + 'static,
+        f: impl Into<Box<dyn FnMut(E) -> ConnectFuture + Send + 'static>>,
```

Comment:
It's weird to me that the thing to me that's called `endpoint` is the socketaddr but the the thing that is an `Endpoint` is called `f` -- when I saw `endpoint.into()` i assumed it was converting it to a connect... Can we clarify naming?